### PR TITLE
Support changing the MPM using a new HTTPD_MPM environment variable.

### PR DESCRIPTION
**Pull request by notroj**

### CODE REVIEW

File: 2.4/root/usr/share/container-scripts/httpd/README.md [hunk 1]
```diff
@@ -80,6 +80,8 @@ The Apache HTTP Server container image supports the following configuration vari
 **`HTTPD_LOG_TO_VOLUME`**  
        By default, httpd logs into standard output, so the logs are accessible by using the docker logs command. When `HTTPD_LOG_TO_VOLUME` is set, httpd logs into `/var/log/httpd24`, which can be mounted to host system using the container volumes. This option is only allowed when container is run as UID 0.
 
+**`HTTPD_MPM`**
+       The variable `HTTPD_LOG_TO_VOLUME` can be set to change the default Multi-Processing Module (MPM) from the package default MPM.
```
Comment (omron93) [inline 2.4/root/usr/share/container-scripts/httpd/README.md]: Should this really be `HTTPD_LOG_TO_VOLUME`?

**Comment by omron93** [comment]
[test][test-openshift]

File: 2.4/root/usr/share/container-scripts/httpd/README.md (hunk 1 above)
Comment (notroj) [reply 2.4/root/usr/share/container-scripts/httpd/README.md]: Good catch, yes, thanks.

**Comment by omron93** [comment]
[test][test-openshift]

**Comment by omron93** [comment]
> Not sure what/where I need to make changes in common/, can you give me some pointers? I don't see similar image-specific stuff in common/

[common/](https://github.com/sclorg/container-common-scripts) directory contains build and test code which is common for all images.

Problem revealed by centos7-openshift test is related to https://github.com/sclorg/container-common-scripts/pull/92 . Newer versions of `oc` client changed parameter name.

So common/ directory in nginx-container have to be updated to reflect this change. And it's better to do it in this PR, to be able to test this change. Thanks.

So please do:
```
git submodule update --remote
git add common
git commit -m "Update common/ submodule"
```

**Comment by notroj** [comment]
@omron93 Sorry, didn't understand that first time.  Got it, done now.

**Comment by notroj** [comment]
@omron93 How's it look now?

**Comment by pkubatrh** [comment]
[test][test-openshift]

**Comment by dustymabe** [comment]
relevant comment: https://bugzilla.redhat.com/show_bug.cgi?id=1586332#c8

**Comment by omron93** [comment]
It looks good, thanks. Code in common/ was wrong (is fixed now - https://github.com/sclorg/container-common-scripts/pull/93). But I'm going to fix fedora image in #63.  So IMHO it's not needed to update submodule now - it's updated in #63 .

Other tests are passing and no regression is introduced. Merging.

**Comment by dustymabe** [comment]
> relevant comment: https://bugzilla.redhat.com/show_bug.cgi?id=1586332#c8

anyone think this would be a valid thing to do ^^. I think it could help us out a lot

**Comment by omron93** [comment]
@dustymabe Sorry, I've missed you comment. I hope I understand it right and your proposal is and extension of this change. So if your proposal is accepted, new PR can be created and no reversion of this PR is needed. 

About a question, I guess it's question to: @notroj @uhliarik @remicollet ?

**Comment by dustymabe** [comment]
> I hope I understand it right and your proposal is an extension of this change. So if your proposal is accepted, new PR can be created and no reversion of this PR is needed.

that is correct

> About a question, I guess it's question to: @notroj @uhliarik @remicollet ?

ok, thanks

**Pull request merged**